AmplifyPlugins/Analytics/Sources/AWSPinpointAnalyticsPlugin/Support/Utils/AnalyticsErrorHelper.swift

```diff
@@ -14,6 +14,8 @@ enum AnalyticsErrorHelper {
         switch error {
         case let error as AnalyticsErrorConvertible:
             return error.analyticsError
+        case let error as AuthError:
+            return .configuration(error.errorDescription, error.recoverySuggestion, error)
```

Comment:
One can argue that this isn't really a `.configuration` error, but it's also not really an `.unknown` one. And unfortunately those are the only two options 😅